workload_generator/src/workload_gen.py

```diff
@@ -202,41 +208,16 @@ def performRequest(self, pid, transactionNumber, command, user=None, stockSymbol
             request['amount'] = amount
         if filename:
             request['filename'] = filename
-                
-        try:
-            packetQ.put(request)
-        except socket.error as err:
-            logging.error(f"Queue Put failured with: {err}")
-            sys.exit()
 
-def consumerThread():
-    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-    s.connect((serverAddress, serverPort))
+        requestString = serverURL + reqType
 
-    while True:
-        try:
-            # set get to throw exception if no packet in 5 seconds
-            request = packetQ.get(True, 2)
-        except:
-            break
-
-        requestStr = str(request).encode()
-
-        s.sendall(requestStr.ljust(256))
-
-        packetQ.task_done()
-    
-    time.sleep(2)
-    s.shutdown(socket.SHUT_RDWR)
-    s.close()
-    print("Consumer Thread Finished!")
+        r = requests.post(requestString, json=command_dict)
```

Comment:
HTTP request posted to server with:

http://localhost:8000/api/stocks/ 
or 
http://localhost:8000/api/accounts/

keeping old cmdDict